### PR TITLE
make the about dialog smaller

### LINKED ISSUE #1051: Improve UX in About Window
### Role

I use PolyMC

### Suggestion

The PolyMC About window is very tall and the `About Qt` button is all the way at the bottom left. It should be somewhere at the top, maybe as a new tab

### Benefit

Some people with high DPI scaling or generally small displays can't see the `About Qt` button and have difficulties providing that information in support & issues therefore.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

image
Mockup:
PolyMC-about-ui-edit

### PR DESCRIPTION
**Pull request by DioEgizio**
closes #1051

### CODE REVIEW

File: launcher/ui/dialogs/AboutDialog.ui [hunk 1]
```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>783</width>
-    <height>843</height>
+    <width>573</width>
+    <height>601</height>
```
Comment (TayouVR) [inline launcher/ui/dialogs/AboutDialog.ui]: 🗿 
```
    <width>550</width>
    <height>600</height>
```

Comment (DioEgizio) [reply launcher/ui/dialogs/AboutDialog.ui]: 🗿

**Review by flowln — APPROVED** [review]
qt dumdum

**Review by timoreo22 — APPROVED** [review]
Big change for Small window

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**